.travis.yml

```diff
@@ -6,6 +6,15 @@ python:
   - 3.5
   - 3.6-dev
 
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode6.4
+      language: generic
+      python: 3.5
+      env:
+        - DYLD_LIBRARY_PATH=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/
+
```

Comment:
Just add the directories you want to cache as done in the following file. 

https://github.com/NIT-dgp/UIP/blob/master/.travis.yml